hack/cluster-openshift.sh

```diff
@@ -289,7 +317,7 @@ if [[ -f "${OS_ISTIO_OC_EXE_PATH}" ]]; then
   fi
 else
    echo "Downloading binary to ${OS_ISTIO_OC_EXE_PATH}"
-   wget -O ${OS_ISTIO_OC_EXE_PATH} ${OS_ISTIO_OC_DOWNLOAD_LOCATION}
+   eval ${DOWNLOADER} ${OS_ISTIO_OC_EXE_PATH} ${OS_ISTIO_OC_DOWNLOAD_LOCATION}
```

Comment:
Just above this line is where you can check if DOWNLOADER is valid and if not then abort.